DataFormats/SiPixelDigiSoA/interface/SiPixelDigisHost.h

```diff
@@ -23,8 +23,8 @@ class SiPixelDigisHost : public PortableHostCollection<SiPixelDigisSoA> {
   uint32_t nDigis() const { return nDigis_h; }
 
 private:
-  uint32_t nModules_h = 0;
   uint32_t nDigis_h = 0;
+  uint32_t nModules_h = 0;
```

Comment:
any reason to reorder these ?

Reply:
I think it the compiler was complaining because of the constructor above and `nDigis_h` being initialized before `nModules_h`.